pacman/executor/algorithm_classes/external_algorithm.py

```diff
@@ -20,6 +20,7 @@ def __init__(
             self, algorithm_id, required_inputs, optional_inputs, outputs,
             required_input_tokens, optional_input_tokens,
             generated_output_tokens, command_line_arguments):
+        # pylint: disable=too-many-arguments
```

Comment:
seriously? if theres a limit to arguments, then maybe we should either consider reducing the length, or asking why pylint has such a limit. Given having that all over the place i feel is almsot as bad as having the warnings

Reply:
I don't think having too many arguments is something that can really be helped....

Reply:
I treat “too many arguments” (and even more so “too many locals”) as a marker for where I need to see if there is an opportunity to refactor. Often it isn't practical, but I'd rather have the false positives as this is not in a tool that we're using for full enforcement.